src/pages/standup/index.tsx

```diff
@@ -0,0 +1,83 @@
+import { FC, useState } from 'react';
+import Head from '@/components/head';
+import Layout from '@/components/Layout';
+
+import { standupUpdateType } from '@/interfaces/standup.type';
+import { FormatDate } from '@/utils/FormatDate';
+import { LOGIN_URL } from '@/constants/url';
+import useAuthenticated from '@/hooks/useAuthenticated';
+import StandUpContainer from '@/components/standup';
+import { useGetUserQuery } from '@/app/services/userApi';
+import { skipToken } from '@reduxjs/toolkit/dist/query';
+
+const StandUp: FC = () => {
+    const [standupUpdate, setStandupUpdate] = useState<standupUpdateType>({
+        type: 'user',
+        completed: '',
+        planned: '',
+        blockers: '',
+    });
+    const [buttonDisable, setButtonDisable] = useState<boolean>(true);
+
+    const { isLoggedIn, isLoading } = useAuthenticated();
+
+    const { isLoading: isAuthenticating } = useGetUserQuery(skipToken);
+
+    // here FormatDate() is a function that is defined inside the util folder and
+    // is used to calculate currentDate - 1 which returns the date as May 16, 2023
+    // which is getting stored in the yesterdayDate variable
+    const yesterdayDate = FormatDate();
+
+    const handleFormSubmission = (event: React.FormEvent<HTMLFormElement>) => {
+        event.preventDefault();
+        setButtonDisable(false);
+        setStandupUpdate({
+            type: 'user',
+            completed: '',
+            planned: '',
+            blockers: '',
+        });
```

Comment:
standup API not implemented right ? If its not implemented then PUT behind the feature flag.

Reply:
to do...